Core/Core/Extensions/BundleExtensions.swift

```diff
@@ -27,7 +27,7 @@ public extension Bundle {
     static let teacherBundleID = "com.instructure.ios.teacher"
     static let parentBundleID = "com.instructure.parentapp"
     static let coreBundleID = "com.instructure.core"
-    static let studentUITestsBundleID = "com.apple.test.StudentUITests-Runner"
+    static let studentUITestsBundleID = "com.instructure.StudentUITests.xctrunner"
```

Comment:
I assume `teacherUITestsBundleID` is going to need a similar change but I haven't ran teacher ui tests yet.